nautobot/core/views/viewsets.py

```diff
@@ -9,8 +9,11 @@ class NautobotUIViewSet(
     mixins.ObjectBulkDestroyViewMixin,
     mixins.ObjectBulkCreateViewMixin,
     mixins.ObjectBulkUpdateViewMixin,
+    mixins.ObjectChangeLogViewMixin,
+    mixins.ObjectNotesViewMixin,
```

Comment:
Does this pose any issue for existing plugins that are using the 1.4 version of `NautobotUIViewSet` and separately implementing the changelog and notes views as was previously needed? Want to make sure we're not doing something backwards-incompatible.

Reply:
I just tested with Providers (by adding back in the path to urls.py, and it doesn't break anything. I am not sure which view it is loading... But nothing broke.